packages/di/src/lib/injectable.ts

```diff
@@ -7,7 +7,7 @@ export const INJECTABLES = new WeakMap<object, Injector>();
 export function injectable<T extends ProviderToken<any>>(Base: T, _?: unknown) {
   return class InjectableNode extends Base {
     constructor(..._: any[]) {
-      super();
+      super(..._);
 
```

Comment:
Since we are actually using this now rename _ to args.
Other than that it looks good!

Reply:
I will... but let me actually run the tests too...

Reply:
Ok I think it ran the tests on-commit

Reply:
Thanks! Tests are also run whenever you open a PR